text-output.txt

```diff
@@ -0,0 +1,21 @@
+TypeError: Cannot read properties of undefined (reading 'provider')
```

Comment:
Is this file required or could be added to `.gitignore`?

Reply:
oops, sorry, was debugging something